src/components/styledComponents.jsx

```diff
@@ -0,0 +1,35 @@
+import styled, { createGlobalStyle } from "styled-components";
+import reset from 'styled-reset';
+
+const GlobalStyle = createGlobalStyle`
+  ${reset};
+`;
```

Comment:
This is just for my own edification: The global style clears the way for our styles....